docs/testplan/PFC_Snappi_Additional_Testcases.md

```diff
@@ -0,0 +1,65 @@
+|           03 | CONGESTION               | **DETECT CONGESTION WITH MISMATCHED INGRESS AND EGRESS:**<br> Aim of the test is ensure that congestion with ingress 400Gbps link and single 100Gbps egress link and mix of lossless and lossy traffic.No losses for lossless priorities (Priority 3 and 4) traffic streams.                                              | - Single 400Gbps Ingress and single 100Gbps egress <br>- Repeat with IPv6 <br>- 1024B packet-size                                                                                                                                      | - 90% of 400Gbps - Lossless Priority 3 and 4<br>- 10% of 400Gbps - Lossy Background Flow [Priority 0, 1, and 2]                                                                                                                                                        | - Single LC - Multiple ASIC <br>- Multiple LC                                    | 1. The single 100Gbps egress should see 100Gbps lossy+lossless traffic<br>2. The 40Gbps lossy traffic should be continue to flow.<br>3. The traffic transmitter port should be receiving PFCs for the lossless priority traffic.<br>4. On receiving the PFCs, the IXIA-ingress should reduce the lossless traffic to around 60Gbps.<br>5. No drop for lossy and lossless traffic.                                                                                                              |
+|           04 | CONGESTION               | **DETECT CONGESTION WITH REAL-LIFE TRAFFIC PATTERN - 90% LOSSLESS and 10% LOSSY:**<br>Aim of the testcase is to ensure that congestion is detected with two ingress and single egresses of same speed and mix of lossless and lossy traffic. No losses for lossless priority traffic (Priority 3 and priority 4 streams). | - Two 100Gbps Ingress and single 100Gbps egress <br>- Repeat test with 2x400Gbps ingress and 1x400Gbps egress <br>- IMIX packet-profile <br>- Unequal traffic lossy and lossless priorities.                                           | - 90% - Lossless Priority 3 and 4 per ingress LC <br>- 10% - Lossy Background Flow [Priority 0, 1, and 2] per ingress LC.                                                                                                                                              | - Single LC - Multiple ASIC <br>- Single LC - Single ASIC <br>- Multiple LC      | 1. No packet drops for Lossy Traffic. <br>2. PFCs for lossless traffic sent from DUT ingress to traffic transmitter. No packet drops for losslesss traffic. <br>3. Expected throughput of (Lossless + lossy traffic only) on egress interface line rate of DUT.                                                                                                                                                                                                                                |

```

Comment:
All traffic should be ip/udp for effective load balancing across the fabric.
We will have cisco-8000 specific adaptations on the size of individual flows.